src/Test/Scene/tPhysicsRuntimeManualTransform.ts

```
@@ -0,0 +1,156 @@
+        constraint.setLinearLowerLimit(new Vector3(0, 0, 0));
+        constraint.setLinearUpperLimit(new Vector3(0, 0, 0));
+        constraint.setAngularLowerLimit(new Vector3(Math.PI / 4, 0, 0));
+        constraint.setAngularUpperLimit(new Vector3(0, 0, 0));
+
+        runtime.addConstraint(constraint, true);
+
+        let time = 0;
+        runtime.onTickObservable.add(() => {
+            box1RigidBody.getTransformMatrixToRef(matrix);
+            matrix.getTranslationToRef(box1.position);
+            Quaternion.FromRotationMatrixToRef(matrix, box1.rotationQuaternion!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)